### PR TITLE
Missing english strings

### PR DESCRIPTION
**Pull request by gienkov**
I've missed two in respawn sorry!

### CODE REVIEW

File: addons/respawn/stringtable.xml [hunk 1]
```diff
@@ -220,6 +221,7 @@
             <Russian>Сообщения об огне по своим</Russian>
         </Key>
         <Key ID="STR_ACE_Respawn_FriendlyFire_Module_Description">
+			<English>Using this module in your mission will make it so any friendly fire kills will be displayed in form of message in chat.</English>
```
Comment (commy2) [inline addons/respawn/stringtable.xml]: TAB ALERT - TAB ALERT - TAB ALERT - TAB ALERT

File: addons/respawn/stringtable.xml [hunk 2]
```diff
@@ -220,6 +221,7 @@
             <Russian>Сообщения об огне по своим</Russian>
         </Key>
         <Key ID="STR_ACE_Respawn_FriendlyFire_Module_Description">
+            <English>Using this module in your mission will make it so any friendly fire kills will be displayed in form of message in chat.</English>
```
Comment (bux578) [inline addons/respawn/stringtable.xml]: in form of "a" message

File: addons/respawn/stringtable.xml [hunk 3]
```diff
@@ -201,6 +201,7 @@
             <Russian>Удалять трупы игроков после дисконнекта?</Russian>
         </Key>
         <Key ID="STR_ACE_Respawn_Module_Description">
+            <English>This module allows you to setup respawn in your mission.</English>
```
Comment (bux578) [inline addons/respawn/stringtable.xml]: This is misleading. To setup respawn you have to use a combination of markers and/or respawn templates in description.ext. It's more like: "This module allows you to enable certain ACE functionality on respawn." (But please find better words, it's early in the morning atm :smile: )

Comment (MikeMatrix) [inline addons/respawn/stringtable.xml]: `This module enables you to configure ACE functionality specific to respawns.`?

**Comment by bux578** [comment]
Thanks @gienkov

**Pull request merged**